data/json/vehicle_parts.json

```diff
@@ -5723,5 +5723,16 @@
             {"item": "plastic_chunk", "count": [1, 3]},
             {"item": "scrap", "count": [2,6]}
         ]
+    },{
+		"id": "car_headlight",
+		"type":"GENERIC",
+		"name": "car headlight",
+		"description": "A vehicle headlight to light up the way.",
+		"symbol": ",",
+		"color": "white",
+		"material": ["glass", "plastic"],
```

Comment:
LED version wouldn't be that fragile. Also, car headlights generally have a plastic casing.

Currently glass (including partially glass) items are so fragile that zombies can break them when walking over them.